docs/DeveloperGuide.md

```diff
@@ -320,121 +320,193 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+
+**MSS**
+1. User requests to clear the list.
+1. GrAB3 shows the updated list.
+Use case ends.
 
 ### Client List
 
-**UC02: Add a client to client list**
+#### UC001: Add a client to client list
 
 **Preconditions: A client must exist**
 
 **Guarantees: A new client will added to the client list** 
 
 **MSS**
-
 1.  User requests to add a new client into the client list.
-2.  GrAB3 shows the updated client list
-
-    Use case ends.
+1.  GrAB3 shows the updated client list
+Use case ends.
 
 **Extensions**
 
 * 1a. Client description is empty.
-
     * 1a1. GrAB3 shows an error message.
-    
-        Use case resumes at step 2.
+    Use case resumes at step 2.
 
-**UC03: Delete a client**
+#### UC003: Delete a client**
 
 **Preconditions: client schedule must exist**
 
 **Guarantees: The client specified by the user will be deleted**
 
 **MSS**
-
 1.  User requests to list all clients.
-2.  GrAB3 displays a list of clients.
-3.  User requests to delete a specific client from the list generated.
-4.  GrAB3 shows the updated list after deletion of the client.
-
-    Use case ends.
+1.  GrAB3 displays a list of clients.
+1.  User requests to delete a specific client from the list generated.
+1.  GrAB3 shows the updated list after deletion of the client.
+Use case ends.
 
 **Extensions**
-
 * 3a. The specified client does not exist.
-
     * 3a1. GrAB3 shows an error message.
+    Use case resumes at step 2.
 
-      Use case resumes at step 2.
-
-**UC04: Edit a client**
+#### UC004: Edit a client
 
 **Preconditions: Client exists in TaskSchedule**
 
 **Guarantees: Client specified by user gets edited**
 
 **MSS**
-
 1.  User requests to list all clients.
-2.  GrAB3 shows a list of clients.
-3.  User requests to edit the description of a specific client from that list.
-4.  GrAB3 shows the updated client.
-
-    Use case ends.
+1.  GrAB3 shows a list of clients.
+1.  User requests to edit the description of a specific client from that list.
+1.  GrAB3 shows the updated client.
+Use case ends.
 
 **Extensions**
-
 * 3a. The given index is invalid.
-
     * 3a1. GrAB3 shows an error message.
+    Use case resumes at step 2.
 
-        Use case resumes at step 2.
-
-
-### Expense List
+### Appointment List
 
-**UC04: Sort expenses by ascending/descending order**
+####UC01: Add an appointment into the appointment list
 
-**Preconditions: Expenses exist in expense list**
+**Preconditions: Appointment list must exist**
 
-**Guarantees: View of expenses sorted in order specified by user **
+**Guarantees: A new appointment will be added to the appointment list**
 
 **MSS**
-
-1.  User requests to list all expenses.
-2.  GrAB3 shows a list of expenses.
-3.  User requests to sort the expenses by ascending order.
-4.  GrAB3 shows the sorted list view.
-
-    Use case ends.
+1.  User adds appointment into appointment list.
+1.  GrAB3 shows the updated list of appointments.
+Use case ends.
 
 **Extensions**
+* 2a. The specified module code is invalid.
```

Comment:
*service

Reply:
Will change this but likely this will not be in used because appointment use case not done